src/devnet-utils.ts

```diff
@@ -43,6 +43,13 @@ export interface IncreaseTimeResponse {
     timestamp_increased_by: number;
 }
 
+export interface GetPredeployedAccountsResponse {
```

Comment:
You can change the name of this interface to `PredeployedAccount` (singular, not plural)

Reply:
Fixe bug #eb4a7c4

Reply:
Fix bug #125

Reply:
[eb4a7c4](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/125/commits/eb4a7c43abf7b559bb9760468b4f130d020c3519)